idseq_dag/engine/pipeline_step.py

```diff
@@ -195,10 +190,10 @@ def wait_until_all_done(self):
             self.upload_thread.join()
         except InvalidInputFileError as e:
             self.update_status_json_file("user_errored")
-            raise e # Raise again to be caught in PipelineFlow and stop other steps
+            raise e  # Raise again to be caught in PipelineFlow and stop other steps
         except Exception as e:
```

Comment:
Similar to my earlier comment, I consider this line to be a bug, one frequently induced by the linter.   It should probably be just a simple "except:  update_status(); raise" as that would be the safest.

Reply:
The only change I introduced in this line is an extra space.